src/server/index.js

```diff
@@ -84,7 +84,13 @@ app.prepare()
             app.render(req, res, "/dashboard", undefined);
         });
 
-        server.get("*", keycloakClient.checkSso(), (req, res) => {
+        // URL paths that might appear in the browser address bar should match this route.
+        const userRouteRegexp = /^\/(dashboard|data|apps|analyses|more)/;
```

Comment:
Minor but the client side uses constants for these routes. May be use them here as well ?

Reply:
Good suggestion. I'll check into that today.